modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -59,35 +60,51 @@ endif::openshift-rosa[]
 ifdef::openshift-rosa[]
 ROSA
 endif::openshift-rosa[]
-cluster INFRA ID:
+cluster ID:
 +
-[source,terminal]
+[source,terminal,subs="quotes"]
 ----
-$ rosa describe cluster --cluster=<cluster_name> | grep -i 'Infra ID'
+$ rosa describe cluster --cluster=<cluster_name> | grep -i ID
```

Comment:
@dfitzmau Pretty sure this will need to use `subs="quotes,verbatim"` in order to work once this hits the XML transform on the Portal end - same goes for any code block that uses angle brackets to indicate content that should be replaced.